packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/guides/telecom-telephony-guides.html

```diff
@@ -3,51 +3,45 @@
         <oui-spinner></oui-spinner>
     </div>
 
-    <div class="row" data-ng-if="!$ctrl.loading.init">
-        <div class="col-md-6">
-            <oui-message data-type="info" data-ng-if="!$ctrl.hasGuides()">
-                <span data-translate="telephony_guides_empty"></span>
-            </oui-message>
+    <div data-ng-if="!$ctrl.loading.init">
```

Comment:
As the guides list is the only things displayed on the page, I remove `row` and `col-md-6`